deploy/templates/bicep/Create-IomtContainerAppEnv.ps1

```diff
@@ -0,0 +1,114 @@
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(3,16)]
+    [ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Service name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$baseName,
+    [Parameter(Mandatory = $true)]
+    [ValidateSet(
+        'australiaeast', 
+        'canadacentral', 
+        'eastus', 
+        'eastus2', 
+        'japaneast', 
+        'koreacentral', 
+        'northcentralus',
+        'northeurope',
+        'qatarcentral',
+        'southcentralus',
+        'southeastasia',
+        'swedencentral',
+        'switzerlandnorth',
+        'westcentralus',
+        'westeurope',
+        'westus2',
+        'westus3',
+        'uksouth'
+   )]
+    [string]$location,
+    [Parameter(Mandatory = $true)]
+    [ValidateSet(
+        'Create',
+        'Lookup',
+        'LookupWithEncounter'
+    )]
+    [string]$resourceIdentityResolutionType
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+# Get current Az context
+try {
+    Write-Host "Get current Az context..."
+    az account show 
+} 
+catch {
+    throw "Please log in with az login cmdlet before proceeding"
+}
+
+# Get current account context - User/Service Principal
+$azAccountId = az account show --query user.name --output tsv
+$azAccountType = az account show --query user.type --output tsv
+if ($azAccountType -eq "user") {
+    Write-Host "Current account context is user: $($azAccountId)"
+    
+    $currentUser = az ad user show --id $azAccountId #Get-AzADUser -UserPrincipalName $azContext.Account.Id 
+
+    if ($currentUser) {
+        $currentObjectId = az ad user show --id $azAccountId --query id --output tsv
+    }
+
+    if (!$currentObjectId) {
+        throw "Failed to find objectId for signed in user"
+    }
+}
+elseif ($azAccountType -eq "servicePrincipal") {
+    Write-Host "Current account context is service principal: $($azAccountId)"
+    $currentObjectId = az ad sp show --id $azAccountId --query id --output tsv
+}
+else {
+    Write-Host "Current context is account of type '$($azAccountType)' with id of '$($azAccountId)"
+    throw "Running as an unsupported account type. Please use either a 'User' or 'Service Principal' to run this command"
+}
+
+# Create a resource group in provided subscription if it doesn't exist and deploy Azure resources needed to run IoMT Service.
+$setupTemplate = "Main.bicep" 
```

Comment:
the letter case in the filename doesn't match. Please make sure it's named "Main.bicep"